packages/node_modules/@node-red/nodes/core/function/90-exec.html

```diff
@@ -53,6 +54,7 @@
         defaults: {
             command: {value:""},
             addpay: {value:false},
+            addpayValue: {value:"payload"},
```

Comment:
Looking at this again, we should try to reuse the `addpay` property rather than add another property.

`addpay` will be set to `true` or `false` in existing nodes. I think we can migrate to this property containing the name of the message property to use - with it set to an empty string if the checkbox is disabled.

I will implement this change myself so we can get it into 1.3.